jobs/hurricane/stats/JEVS_HURRICANE_STATS

```diff
@@ -21,9 +21,10 @@ export jlogfile=${jlogfile:-${DATA}/jlogfile.${jobid}}
 # Specify NET and RUN Name and model
 ####################################
 export NET=${NET:-evs}
-export COMPONENT=${COMPONENT:-define_in_ecf}
-export RUN=${RUN:-define_in_ecf}
-export STEP=${STEP:-stats}
+export COMPONENT=${COMPONENT:-hurricane}
+export RUN=${RUN:-global_det}
+export STEP=${STEP:-stats} 
+export VERIF_CASE=${VERIF_CASE:-tropcyc}   
 
```

Comment:
For completeness, I think it would help if we can modify these 3 lines to look like this:
`export RUN=${RUN:-global_det}  # global_det, global_ens, regional`
`export STEP=${STEP:-plots} # plots, stats`
`export VERIF_CASE=${VERIF_CASE:-tropcyc} # tropcyc, tcgen`
The thought being that future users unfamiliar with the code may know the potential values for those variables.